src/test/java/org/opensearch/security/dlic/rest/api/UserApiTest.java

```diff
@@ -136,6 +138,38 @@ public void testParallelPutRequests() throws Exception {
         deleteUser("test1");
     }
 
+    @Test
```

Comment:
In general, a separate test case for each of the checks would be more appropriate, but as added them all in one test case as that is how the other tests on the class are setup and I'm not sure if there is some underlying reason for that.

Reply:
There is no particular reason that I am aware of. I believe an initial choice was made and then everyone has copied the pattern since. If you are able to make individual tests that would be excellent.